js/controller.js

```diff
@@ -268,11 +270,15 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
       this.accessibilityControls = new AccessibilityControls(this); //keyboard support
       this.state.screenToShow = CONSTANTS.SCREEN.INITIAL_SCREEN;
+
+      if (this.videoVr) {
+        this.mb.subscribe(OO.EVENTS.VR_DIRECTION_CHANGED, 'customerUi', _.bind(this.setViewingDirection, this));
```

Comment:
We don't subscribe to any other events in `onPlayerCreated`. Would it be better to handle this on `init()` or `subscribeBasicPlaybackEvents`?

Reply:
\+